docs/intro.rst

```diff
@@ -123,10 +123,17 @@ Installing to Linux
 
 * Download the ``pywbem_os_setup.sh`` script from one of these sources:
 
-  - :download:`pywbem_os_setup.sh <../pywbem_os_setup.sh>` on this site
-
-  - `pywbem_os_setup.sh <https://raw.githubusercontent.com/pywbem/pywbem/master/pywbem_os_setup.sh>`_
-    on the `master` branch of the pywbem Git repository
+  - :download:`pywbem_os_setup.sh <../pywbem_os_setup.sh>` - this version of
+    the script fits exactly to the pywbem version of this documentation.
+    However, its URL contains a UUID generated by the RTD site, so the URL is
+    not predictable for future pywbem versions.
```

Comment:
ADD: and can break tools that automate the installation process.

Reply:
done